src/app/Http/Controllers/CrudFeatures/AjaxTable.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 namespace Backpack\CRUD\app\Http\Controllers\CrudFeatures;
+namespace LiveControl\EloquentDataTable\DataTable;
```

Comment:
Unless I'm seeing things, is there two namespace declarations here?

I think that is meant to be **use** not **namesapce**